.gitignore

```diff
@@ -109,3 +109,6 @@ dist
 
 # Mac
 .DS_Store
+
+# Npm
+utils.js
```

Comment:
Just a thought: shouldn't we `.gitignore` the `/dist` directory too?

Reply:
Already in the file :)

Reply:
Great! And, sorry, I forgot to look at the entire content :)